packages/zipkin/src/tracer/record.js

```diff
@@ -0,0 +1,13 @@
+class Record {
```

Comment:
Could be an idea, but I imagine the implementations will diverge somewhat in the longer run.